docs/BestPractices.md

```diff
@@ -0,0 +1,44 @@
+#  AWS Java SDK 2.x Best Practices
+
+Here are the best practices of using AWS Java SDK 2.x.
+
+### Reuse SDK client if possible
+
+Each SDK client maintains many resources such as connections and threads, so it is recommended to share a single instance of the client to avoid the overhead of resources management. All SDK clients are thread safe. 
```

Comment:
I think we can make this a little more clear. The reason for resource pooling is because creating them is expensive. As it's written, it sounds like the managing them is the expensive part 😛

Reply:
Yeah, will update.